pom.xml

```diff
@@ -93,6 +92,24 @@
       <artifactId>docker-fixtures</artifactId>
       <version>1.0</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <!-- provided by jenkins-core -->
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+        </exclusion>
+        <exclusion>
+          <!-- provided by jenkins-core -->
+          <groupId>org.jenkins-ci</groupId>
+          <artifactId>annotation-indexer</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.sshd</groupId>
+      <artifactId>sshd-core</artifactId>
+      <version>1.6.0</version>
+      <scope>test</scope>
```

Comment:
I do not get that. Jenkins core ships its own version. By using it you make the test environment different from the real one. Why?

Reply:
[See previous comment](https://github.com/jenkinsci/ssh-slaves-plugin/pull/76#discussion_r154013397)